### PR TITLE
onionshare: fix dependencies

### PR DESCRIPTION
**Pull request by mrsmith0x00**
Fix for #2028.

### CODE REVIEW

File: packages/python-stem/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+makedepends=('git' 'python-setuptools')
+source=('git://git.torproject.org/stem.git')
+sha1sums=('SKIP')
+
+pkgver() {
+  cd "$srcdir/stem"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
```
Comment (FFY00) [inline packages/python-stem/PKGBUILD]: While you're at it, you can also add a build step.

Comment (mrsmith0x00) [reply packages/python-stem/PKGBUILD]: Added.

**Comment by mrsmith0x00** [comment]
I apologize that hasn't corrected at once.

**Comment by noptrix** [comment]
thank you guys.

**Pull request merged**